packages/ui-components/src/components/form/RadioGroup.vue

```diff
@@ -18,6 +18,7 @@
               : 'hover:border-outline-1'
           ]"
           :disabled="disabled || option.disabled"
+          type="button"
```

Comment:
a button w/o an explicit type gets implicitly treated as a submit button in `<form/>`